### PR TITLE
Added Steins_gate

### PR DESCRIPTION
**Pull request by ibnufachrizal**

### CODE REVIEW

File: src/store/listAnime.js [hunk 1]
```diff
@@ -300,5 +300,22 @@ export default [
...
+        animeEmoji:"🤵🏼👊🏼👱🏼",
+        animeName:"Kimetsu_no_Yaiba",
+        animeYear:2019,
+        animeLink:"https://myanimelist.net/anime/38000/Kimetsu_no_Yaiba"
+	{
+        animeEmoji:"🚴️👦️👓️",
+        animeName:"Steins;Gate",
+        animeYear:2011,
+        animeLink:"https://myanimelist.net/anime/9253/Steins_Gate"
+    },
```
Comment (2pai) [inline src/store/listAnime.js]: why using a comma? this cause an error

**Comment by 2pai** [comment]
please 1 pr , 1 anime

File: src/store/listAnime.js (hunk 1 above)
Comment (ibnufachrizal) [reply src/store/listAnime.js]: Oh sorry

**Comment by ibnufachrizal** [comment]
how to delete branch?

**Pull request closed without merging**